packages/protocol-kit/src/Safe.ts

```diff
@@ -421,13 +455,37 @@ class Safe {
     return this.#moduleManager.isModuleEnabled(moduleAddress)
   }
 
+  /**
+   * Checks if a specific address or passkey is an owner of the current Safe.
+   *
+   * @param owner - The owner address or a passkey object
+   * @returns TRUE if the account is an owner
+   */
+  async isOwner(owner: string | passkeyArgType): Promise<boolean> {
+    const isOwnerAddress = typeof owner === 'string'
```

Comment:
The `this.#safeProvider` has an util to detect if this is an `address`. Consider to just query it. I dont think it will match eip3770 though.

Reply:
In this case we need to know if the `owner` param is a `string` or a `passkeyArgType` not an `address`.